Sprint-1/interpret/time-format.js

```diff
@@ -1,24 +1,58 @@
-const movieLength = 8784; // length of movie in seconds
+const movieLength = 8784; // length of movie in seconds 8784
 
 const remainingSeconds = movieLength % 60;
+console.log(remainingSeconds);
+
 const totalMinutes = (movieLength - remainingSeconds) / 60;
+console.log(totalMinutes);
 
 const remainingMinutes = totalMinutes % 60;
+console.log(remainingMinutes);
+
+
 const totalHours = (totalMinutes - remainingMinutes) / 60;
 
 const result = `${totalHours}:${remainingMinutes}:${remainingSeconds}`;
 console.log(result);
 
-// For the piece of code above, read the code and then answer the following questions
+/*
+    For the piece of code above, read the code and then answer the following questions
+
+    a) How many variable declarations are there in this program?
+
+        -------- 6 in total
+        -------- movieLength - declared as const
+        -------- remainingSeconds - declared as const
+        -------- totalMinutes - declared as const
+        -------- remainingMinutes - declared as const
+        -------- totalHours - declared as const
+        -------- result - declared as const
+
+
+    b) How many function calls are there?
+
+        -------- console.log(result);
+
+    c) Using documentation, explain what the expression movieLength % 60 represents
+
+        -------- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder
+        -------- Remainder:
+        -------- WITH % we can obtain the remainder between the var movieLength % 60; and stored in remainingMinutes
 
-// a) How many variable declarations are there in this program?
+    d) Interpret line 4, what does the expression assigned to totalMinutes mean?
 
-// b) How many function calls are there?
+        -------- This line is taking the leftover remainder in remainingMinutes and leaving just the
+            total amount that we can divide perfectly in whole number to see how many minutes we have.
```

Comment:
Try clarifying that this line converts the difference (after subtracting remainingSeconds from movieLength) into minutes by dividing by 60, rather than focusing on 'leftover remainder' or 'perfect division'.